src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/handlers/presubmit/PaymentAppealPreparer.java

```diff
@@ -178,11 +179,13 @@ private boolean isLegalRepJourney(AsylumCase asylumCase) {
     }
 
 
-    private boolean isHuEaEuPa(AsylumCase asylumCase) {
+    private boolean isHuEaEuPaAda(AsylumCase asylumCase) {
         Optional<AppealType> optionalAppealType = asylumCase.read(APPEAL_TYPE, AppealType.class);
         if (optionalAppealType.isPresent()) {
             AppealType appealType = optionalAppealType.get();
-            return List.of(HU, EA, EU, PA).contains(appealType);
+            boolean isNonAdaOrNonDetained = asylumCase.read(IS_ACCELERATED_DETAINED_APPEAL, YesOrNo.class)
+                .orElse(YesOrNo.NO).equals(YesOrNo.NO);
+            return isNonAdaOrNonDetained && (List.of(HU, EA, EU, PA).contains(appealType));
```

Comment:
Is the name of the variable midleading? Should it be `isNonAda` rather than `isNonAdaOrNonDetained`?

Reply:
Renamed as suggested :)